src/SDKs/Compute/Management.Compute/Generated/IAvailabilitySetsOperations.cs

```diff
@@ -147,7 +166,7 @@ public partial interface IAvailabilitySetsOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<IEnumerable<AvailabilitySet>>> ListWithHttpMessagesAsync(string resourceGroupName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<IPage<AvailabilitySet>>> ListWithHttpMessagesAsync(string resourceGroupName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This will be a breaking change, please bump the major version for nuget package.

Reply:
@dsgouda updated the version to 21.0.0.  Thanks.